tasks/sep/019-fp-holocene-upgrade/NestedSignFromJson.s.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {NestedSignFromJson as OriginalNestedSignFromJson} from "script/NestedSignFromJson.s.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {GnosisSafe} from "safe-contracts/GnosisSafe.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {DisputeGameFactory} from "@eth-optimism-bedrock/src/dispute/DisputeGameFactory.sol";
+import {FaultDisputeGame} from "@eth-optimism-bedrock/src/dispute/FaultDisputeGame.sol";
+import {PermissionedDisputeGame} from "@eth-optimism-bedrock/src/dispute/PermissionedDisputeGame.sol";
+import {MIPS} from "@eth-optimism-bedrock/src/cannon/MIPS.sol";
+import {ISemver} from "@eth-optimism-bedrock/src/universal/ISemver.sol";
+
+contract NestedSignFromJson is OriginalNestedSignFromJson {
+    using LibString for string;
+
+    DisputeGameFactory constant dgfProxy = DisputeGameFactory(0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1);
```

Comment:
This should be read from the superchain registry instead of hardcoding it, for example https://github.com/ethereum-optimism/superchain-ops/blob/9f0a42d94ddbe42b7110c394516c58b390195ca8/tasks/eth/fp-recovery/003-enable-permissionless-game/SignFromJson.s.sol#L55

Reply:
Agreed with @mds1 that this will only be addressed in the follow up task to fix the Sepolia FPs with the correct `DelayedWETH` implementation.